config.c

```diff
@@ -82,14 +84,17 @@ void write_config(config_params_s *conf) {
           conf->init_fullscreen ? "true" : "false");
   sprintf(ini_values[initPointer++], "use_gpu=%s\n",
           conf->init_use_gpu ? "true" : "false");
+  sprintf(ini_values[initPointer++], "wait_for_device=%s\n",
```

Comment:
You could use `snprintf` here instead to avoid zero-termination / buffer overflow problems.